.github/workflows/build.yaml

```
@@ -61,6 +62,21 @@ on:
           - "tenant"
           - "user-otp"
           - "xstate-chatbot"
+          - "workbench-ui"
+          - "vault-enc-service"
+          - "egov-hrms"
+          - "zuul-kubernetes-discovery"
+          - "egov-enc-service"
```

actionlint (1.7.4)

option "egov-enc-service" is duplicated in options of "service" input

(events)